org.plomgrading.PlomClient.json

```diff
@@ -0,0 +1,35 @@
+{
+    "app-id": "org.plomgrading.PlomClient",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.14",
+    "sdk": "org.kde.Sdk",
+    "command": "PlomClient",
+    "finish-args": [
+        "--socket=x11",
```

Comment:
you need --share=ipc as well if you use --socket=x11 (yes, it's irritating, sorry)
for Qt based stuff you often need --device=dri as well for 3d acceleration. Do you see messages about that in local testing? If not, feel free to ignore.

Reply:
Thanks!  I did that and added `wayland` as well.  Personally QT and Wayland don't cooperate super well on my Fedora 31 but I have high hopes for the future!